### ISSUE TITLE
Remove duplicated font family names with a case-insensitive comparison

### ISSUE THREAD

**Issue by 7studio** [TrySound/postcss-minify-font-values#1]
For the moment, the module removes duplicated font family names with a case-sensitive comparison but as defined in the [W3C spec.](http://www.w3.org/TR/css3-fonts/#font-family-casing), font family names are case-insensitive. 

IMHO, it could be interesting to take into account this particularity when the module removes duplicated font family names.

**input**:
```
h1 {
  font-family: Helvetica, Arial, helvetica, sans-serif;
}
```
**output**:
```
h1 {
  font-family: Helvetica, Arial, sans-serif;
}
```
I will be happy to open a PR if this idea interests you.

**Comment by TrySound** [comment]
@7studio Spec is good. Did you try this is browsers?

**Comment by 7studio** [comment]
@TrySound Yes I did :relieved: I would not propose you a feature without testing it. 
You can try by yourself with the help of this test case: http://codepen.io/7studio/pen/qOKrbb

**Comment by TrySound** [comment]
@7studio Nice! Okay, waiting PR.
